modules/JsRiseV2G/index.js

```diff
@@ -62,6 +62,19 @@ function JavaStartedDeferred(mqtt_base_path, module_name) {
   });
 }
 
+function choose_first_ipv6_local() {
+  const all_ifs = os.networkInterfaces();
+
+  const found_if = Object.keys(all_ifs).find(
+    (iface) => all_ifs[iface].filter((info) => info.address.includes('fe80')).length > 0
+  );
+
+  if (found_if) return found_if;
+
+  evlog.warning('No necessary IPv6 link-local address was found!');
+  return 'eth0';
+}
+
 function check_network_interface(network_iface) {
   let networkfound = false;
   const net_init = os.networkInterfaces();
```

Comment:
this you won't need anymore

Reply:
I mean the net_init var

Reply:
if you use single quotes around "auto" codacy should be silent

Reply:
In the case of a HIL, it should still be checked whether the specified interface is available. Therefore I would leave the check_network_interface function with the net_init var in.